monorepo/scripts/utils.js

```diff
@@ -0,0 +1,132 @@
+// @ts-check
+const fs = require('fs');
+const path = require('path');
+const spawn = require('child_process').spawn;
+
+const renameFolder = (oldPath, newPath) => {
+    return new Promise((resolve, reject) => {
+        fs.rename(path.resolve(oldPath), path.resolve(newPath), function(err) {
+            if (err) {
+                reject(err);
+            } else {
+                resolve();
+            }
+        });
+    });
+};
+
+const deleteFolder = folder => {
+    const delPath = path.resolve(folder);
+    if (fs.existsSync(delPath)) {
+        fs.readdirSync(delPath).forEach((file, index) => {
+            const currentPath = path.resolve(delPath, `./${file}`);
+            if (fs.lstatSync(currentPath).isDirectory()) {
+                deleteFolder(currentPath);
+            } else {
+                fs.unlinkSync(currentPath);
+            }
+        });
+        fs.rmdirSync(delPath);
+    }
+};
+
+const readFile = file => {
+    return new Promise((resolve, reject) => {
+        fs.readFile(path.resolve(file), 'UTF8', (err, data) => {
+            if (err) {
+                reject(err);
+            } else {
+                resolve(data);
+            }
+        });
+    });
+};
+
+const readFiles = folder => {
+    return new Promise((resolve, reject) => {
+        fs.readdir(path.resolve(process.cwd(), folder), { withFileTypes: true }, function(
+            err,
+            files
+        ) {
+            if (err) {
+                reject(err);
+            } else {
+                resolve(files);
+            }
+        });
+    });
+};
+
+const writeFile = (file, data) => {
```

Comment:
I really don't get the purpose of it. I wanted to show how simple it is use to use monorepo with fuse, primarily with a focus on efficient configs. This doesn't seem efficient to me ;-)

Reply:
removed, was me getting carried away :-)